test/functional/tests/io_class/test_io_class_directory.py

```diff
@@ -17,11 +17,10 @@
 from .io_class_common import *
 
 
+@pytest.mark.require_disk("cache", [DiskType.optane, DiskType.nand])
+@pytest.mark.require_disk("core", [DiskType.sata, DiskType.hdd])
```

Comment:
this is a limitation for cores - we usually just want cores to be "lower" type than caches, so if we have optanes as caches, we don't want to exclude nands as possible cores, and that's what happened here

Reply:
Done.